src/Model/PrimaryTypeGo.cs

```diff
@@ -104,8 +104,7 @@ public virtual string ImplementationName
                         return "string";
 
                     case KnownPrimaryType.Object:

```

Comment:
@jhendrixMSFT can you regenerate the sdk and check if there is any issue ? I remember I had to change to objects to be `interface{}` to properly handle `AdditionalProperties` (e.g. when the service would return an array as an `object` )

Reply:
I rebuilt the SDK and verified this doesn't result in any regressions.